frontend-react/src/hooks/UseCreateFetch.ts

```diff
@@ -39,20 +39,12 @@ function createTypeWrapperForAuthorizedFetch(
     ): Promise<T> {
         const headerOverrides = options?.headers || {};
         const headers = { ...authHeaders, ...headerOverrides };
-        // this system assumes that we want to be making authenticated
-        // requests whenever possible
-        if (!headers.authorization || headers.authorization.length < 8) {
```

Comment:
I just took this out since it feels redundant.  If we make an unauthenticated request that shouldn't be handled, we'll get an error code anyhow 🤷